### PR TITLE
Update styling for medical conditions page

### PR DESCRIPTION
**Pull request by NickColley**
Bring inline with: https://c19-prototypes.cloudapps.digital/vulnerable-people/v3/q2-conditions

image

### CODE REVIEW

**Review by bilbof — APPROVED** [review]

**Comment by NickColley** [comment]
Tests dont fail locally, scss-lint seems to be failing on CI, we should consider disabling scss-lint if possible since it's not critical.

**Comment by NickColley** [comment]
Will rebase off https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/pull/42 once merged

**Review by injms — APPROVED** [review]

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -1 +1,8 @@
 @import 'govuk_publishing_components/all_components';
+
+.app-list--spaced > li {
+    margin-bottom: govuk-spacing(3);
+}
+.app-list--spaced > li:last-child {
+    margin-bottom: govuk-spacing(0);
+}
```
Comment (huwd) [inline app/assets/stylesheets/application.scss]: might need a new line

**Review by huwd — APPROVED** [review]

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (NickColley) [reply app/assets/stylesheets/application.scss]: Added thanks :))

**Pull request merged**